docs/general/license/transfer.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How to transfer
+sidebar_position: 4
+---
+
+## How to transfer a license to another device{#transfer}
+
+The license key can be activated on a [limited number of devices](../what-is#devices). If the key has already been used on the maximum allowed number of devices, you need to unbind a device from your account. 
+
+### How to unbind a device from a license
+
+There's three ways in which you can do it:
+
+  * by clicking the *Reset license* button (applies to desktop apps)
+  * by logging out of your account in an AdGuard product
+  * by removing the device in your AdGuard account:
+     1. Click **AdGuard Ad Blocker** in your [AdGuard account](https://my.adguard.com/)
+        ![Account Ad Blocker tab *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/general/newaccount-unbind-device-0.png)
+     2. In the license card, click the device you want to unbind
+        ![Select device *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/general/newaccount-unbind-device-1.png)
+     3. Click **Remove**
+        ![Click Remove *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/general/newaccount-unbind-device-2.png) 
+
+
+After you have unbound the number of devices you need, you can log in to your AdGuard account on new devices. The license will be bound to them.
+
+## How to transfer a license to another AdGuard account{#transferaccount}
+
+ 1. Go to the first AdGuard account
+ 2. Go to **Licenses** tab in the menu
+ 3. Find the license you want to transfer and click **Unbind from account**
+    ![Click Unbind from account *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/general/newaccount-transfer-to-account.png)
+ 4. Click **Unbind** (**IMPORTANT**: be sure to note/remember the license key)
```

Comment:
we should abstain from using caps lock for typing important/warning and other alarm messages